evasquez26/angular4-startup
# S5-ONE-144 Request addition to organization

## Story / Task

see [ONE-144](https://bixlabs.atlassian.net/browse/ONE-144)


## What this PR does?

Adds the ability to request to join an organization (creates a role request)


### Before 

You could not request a role from the organizations page.

### After

You can ask to join an organization from the details page.

### How to test / reproduce

- Login to the app
- Go to schools
- Select a school
- See there's a request role button
- You should be able to request any role and fill the information
- You should be able to send the request and see that the request is listed in the page

### Related PRs
N/A

## Migrations

NO

Thank you.
